sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -2721,6 +2707,68 @@ class HiveDDLSuite
     }
   }
 
+  test("SPARK-31828: Retain table properties at CreateTableLikeCommand") {
+    val catalog = spark.sessionState.catalog
+    withTable("t1", "t2", "t3", "t4", "t5", "t6") {
+      sql("CREATE TABLE t1(c1 int) TBLPROPERTIES('k1'='v1', 'k2'='v2')")
```

Comment:
can we add `USING hive`? otherwise if we change the default table format, this test will be broken.

Reply:
added.